docs/content/doc/usage/command-line.en-us.md

```diff
@@ -99,6 +99,13 @@ Admin operations:
         - `--password value`, `-p value`: New password. Required.
       - Examples:
         - `gitea admin user change-password --username myname --password asecurepassword`
+    - `must-change-password`:
+      - Args:
+        - `[username...]`: Users that must change their passwords
+      - Options:
+        - `--all`, `-A`: Force a password change for all users
```

Comment:
Still, why `-A` and not `-a`?
As far as I can see, it's the only sub-command with a capital letter…

Reply:
I guess I thought --all tended to be -A but I don't mind making it -a if others prefer